database/B/Bob Dylan/Rough and Rowdy Ways/Murder Most Foul

```diff
@@ -0,0 +1,211 @@
+
+Play Etta James too, play "I'd Rather Go Blind,"
+play it for the man with the telepathic mind.
+Play John Lee Hooker, play "Scratch My Back."
+Play it for that strip club owner named Jack.
+
+Guitar Slim going down slow,
+play it for me and for Marilyn Monroe.
+Play "Please Don't Let Me Be Misunderstood."
+Play it for the First Lady, she ain't feeling any good.
+Play Don Henley, play Glenn Frey,
+take it to the limit and let it go by.
+Play it for Carl Wilson, too,
+looking far, far away down Gower Avenue.
+
+Play tragedy, play "Twilight Time,"
+take me back to Tulsa to the scene of the crime.
+Play another one and "Another One Bites the Dust."
+Play "The Old Rugged Cross" and "In God We Trust."
+
+Ride the pink horse down that long, lonesome road,
+stand there and wait for his head to explode.
+Play "Mystery Train" for Mr. Mystery,
+the man who fell down dead like a rootless tree.
+
+Play it for the reverend, play it for the pastor,
+play it for the dog that got no master.
+Play Oscar Peterson, play Stan Getz,
+play "Blue Sky," play Dickey Betts.
+
+Play Art Pepper, Thelonious Monk,
+Charlie Parker and all that junk.
+All that junk and "All That Jazz,"
+play something for the Birdman of Alcatraz.
+
+Play Buster Keaton, play Harold Lloyd,
+play Bugsy Siegel, play Pretty Boy Floyd.
+Play the numbers, play the odds.
+Play "Cry Me a River" for the Lord of the gods.
+
+Play Number 9, play Number 6,
+play it for Lindsey and Stevie Nicks.
+Play Nat King Cole, play "Nature Boy,"
+play "Down in the Boondocks" for Terry Malloy.
+
+Play "It Happened One Night" and "One Night of Sin,"
+there's twelve million souls that are listening in.
+Play "Merchant of Venice", play "Merchants of Death,"
+play "Stella by Starlight" for Lady Macbeth.
+
+Don't worry, Mr. President, help's on the way,
+your brothers are comin', there'll be hell to pay.
+Brothers? What brothers? What's this about hell?
+Tell them, "We're waitin', keep comin',"
+we'll get them as well.
+
+Love Field is where his plane touched down,
+but it never did get back up off the ground,
+was a hard act to follow, second to none.
+They killed him on the altar of the rising sun.
+
+Play "Misty" for me and "That Old Devil Moon,"
+play "Anything Goes" and "Memphis in June."
+Play "Lonely at the Top" and "Lonely Are the Brave,"
+play it for Houdini spinning around in his grave.
+
+Play Jelly Roll Morton, play "Lucille,"
+play "Deep in a Dream", and play "Driving Wheel."
+Play "Moonlight Sonata" in F-sharp
+and "A Key to the Highway" for the king on the harp.
+
+Play "Marching Through Georgia" and "Dumbarton's Drums,"
+play darkness and death will come when it comes.
+Play "Love Me or Leave Me" by the great Bud Powell,
+play "The Blood-Stained Banner", play "Murder Most Foul."
+
+
+___________________________________________
+Name              Murder Most Foul
+Artist            Bob Dylan
+Album             Rough and Rowdy Ways
+Track no          1 (CD 2)
```

Comment:
It's better to use `Disc no` metadata key here, this way we could arrange and parse things easier in the future; the full list of all currently accepted metadata keys can be found [here](https://github.com/Lyrics/lyrics/blob/master/tests/database-tests.d/4-metadata.py#L16)